### ISSUE TITLE
[BUG] [Dev branch] Failing to load non vision model

### ISSUE THREAD

**Issue by TyraVex** [theroyallab/tabbyAPI#248]
### OS

Linux

### GPU Library

CUDA 12.x

### Python version

3.12

### Describe the bug

Hello,

I tried the dev branch for the vision feature.

Unfortunately, loading a non vision model using

```shell
  curl 'http://localhost:5000/v1/model/load' \
    -H "x-admin-key: $LOCAL_API_KEY" \
    -H 'Content-Type: application/json' \
    -d '{"model_name":"'"$model"'"}'
```

With model being `Qwen2.5-Coder-32B-Instruct-8.0bpw`, fails: `data: {"error":{"message":"Unknown vision model type: None","trace":null}}`

### Reproduction steps

```shell
  curl 'http://localhost:5000/v1/model/load' \
    -H "x-admin-key: $LOCAL_API_KEY" \
    -H 'Content-Type: application/json' \
    -d '{"model_name":"'"$model"'"}'
```

### Expected behavior

Be able to load non vision models

### Logs

```
ERROR:    Traceback (most recent call last):
ERROR:      File "/home/user/files/ai/tabby/endpoints/core/utils/model.py", line
120, in stream_model_load
ERROR:        async for module, modules, model_type in load_status:
ERROR:      File "/home/user/files/ai/tabby/common/model.py", line 90, in 
load_model_gen
ERROR:        async for module, modules in load_status:
ERROR:      File "/home/user/files/ai/tabby/backends/exllamav2/model.py", line 
557, in load_gen
ERROR:        async for value in iterate_in_threadpool(model_load_generator):
ERROR:      File "/home/user/files/ai/tabby/common/concurrency.py", line 30, in 
iterate_in_threadpool
ERROR:        yield await asyncio.to_thread(gen_next, generator)
ERROR:              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ERROR:      File "/usr/lib/python3.12/asyncio/threads.py", line 25, in to_thread
ERROR:        return await loop.run_in_executor(None, func_call)
ERROR:               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ERROR:      File "/usr/lib/python3.12/concurrent/futures/thread.py", line 58, in
run
ERROR:        result = self.fn(*self.args, **self.kwargs)
ERROR:                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ERROR:      File "/home/user/files/ai/tabby/common/concurrency.py", line 20, in 
gen_next
ERROR:        return next(generator)
ERROR:               ^^^^^^^^^^^^^^^
ERROR:      File 
"/home/user/files/ai/envs/exllama/lib/python3.12/site-packages/torch/utils/_cont
extlib.py", line 36, in generator_context
ERROR:        response = gen.send(None)
ERROR:                   ^^^^^^^^^^^^^^
ERROR:      File "/home/user/files/ai/tabby/backends/exllamav2/model.py", line 
637, in load_model_sync
ERROR:        self.vision_model = ExLlamaV2VisionTower(self.config)
ERROR:                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ERROR:      File 
"/home/user/files/ai/envs/exllama/lib/python3.12/site-packages/exllamav2/vlm/vis
ion_tower.py", line 49, in __init__
ERROR:        raise ValueError(f"Unknown vision model type: 
{cfg.vision_model_type}")
ERROR:    ValueError: Unknown vision model type: None
ERROR:    Sent to request: Unknown vision model type: None
```

### Additional context

It works on the non dev branch.
Built the exllama dev version for tabby dev 

### Acknowledgements

- [X] I have looked for similar issues before submitting this one.
- [X] I have read the disclaimer, and this issue is related to a code bug. If I have a question, I will use the Discord server.
- [X] I understand that the developers have lives and my issue will be answered when possible.
- [X] I understand the developers of this program are human, and I will ask my questions politely.

**Comment by bdashore3** [comment]
This is why the branch is not merged into main. It's being worked on.

**Comment by TyraVex** [comment]
Indeed, I agree. I don't think devs are against beta feedback, though

And if i'm wrong, they can close the issue, it's understandable

**Comment by DocShotgun** [comment]
This error is due to trying to load a non-vision model with vision enabled. Despite `config_sample.yml` noting the default for vision is `False`, from looking at the code it defaults to `True` internally if not provided. API loading of models in general is broken currently on the vision branch anyways, so you will need to load the model on launch via `config.yml` for now.

**Comment by bdashore3** [comment]
Fixed in 0ab393f
